packages/notifications/resources/views/notification.blade.php

```diff
@@ -17,33 +25,47 @@
     ])"
     x-transition:leave-end="scale-95 opacity-0"
 >
-    @if ($icon = $getIcon())
-        <x-filament-notifications::icon :icon="$icon" :color="$getIconColor()" />
-    @endif
-
-    <div class="grid flex-1">
-        @if ($title = $getTitle())
-            <x-filament-notifications::title>
-                {{ str($title)->markdown()->sanitizeHtml()->toHtmlString() }}
-            </x-filament-notifications::title>
+    <div
+        @class([
+            'flex gap-3 w-full',
+            'py-2 pl-6 pr-2' => $isInline,
+            'p-4 rounded-xl border' => ! $isInline,
+            'border-primary-500/50 bg-primary-500/10 dark:bg-primary-500/20' => $color === 'primary',
+            'border-secondary-500/40 bg-secondary-500/10 dark:bg-secondary-500/20' => $color === 'secondary',
+            'border-danger-500/40 bg-danger-500/10 dark:bg-danger-500/20' => $color === 'danger',
+            'border-success-500/40 bg-success-500/10 dark:bg-success-500/20' => $color === 'success',
+            'border-warning-500/40 bg-warning-500/10 dark:bg-warning-500/20' => $color === 'warning',
+            'border-gray-300 dark:border-gray-700' => $color === null,
```

Comment:
Could you refactor this into a match expression?

Reply:
Also, custom color should be supported like elsewhere.

Reply:
Will handle this tomorrow. Thanks.

Reply:
We'll still need to handle custom colors.

Reply:
Done.